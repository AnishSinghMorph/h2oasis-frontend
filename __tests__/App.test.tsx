import { render } from '@testing-library/react-native';
import React from 'react';
import { Text, View } from 'react-native';

// Simple test component
const TestComponent = () => (
  <View>
    <Text>H2Oasis App</Text>
  </View>
);

describe('App Smoke Test', () => {
  it('should render without crashing', () => {
    const { getByText } = render(<TestComponent />);
    expect(getByText('H2Oasis App')).toBeTruthy();
  });

  it('should match snapshot', () => {
    const tree = render(<TestComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
