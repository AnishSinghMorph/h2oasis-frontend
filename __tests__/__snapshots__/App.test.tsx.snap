// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Smoke Test should match snapshot 1`] = `
<View>
  <Text>
    H2Oasis App
  </Text>
</View>
`;
